Source/Core/DolphinQt2/Config/SettingsWindow.cpp

```diff
@@ -36,9 +36,6 @@ SettingsWindow::SettingsWindow(QWidget* parent) : QDialog(parent)
     // Actual Settings UI
     SetupSettingsWidget();
 
-    MakeUnfinishedWarning();
-
-    content_inner->addWidget(m_warning_group);
     content_inner->addWidget(m_settings_outer);
 
     content->addLayout(content_inner);
```

Comment:
Ah, figured it out. QBoxLayout likes to add margins, and setting it to 0 (`->setMargin(0)`) fixes it. I will add the fix in one of my upcoming PRs.